src/routes/stop_monitoring.rs

```diff
@@ -44,6 +44,10 @@ fn create_monitored_stop_visit(
 ) -> model::MonitoredStopVisit {
     let stop = &data.ntm.stop_points[connection.stop_point_idx];
     let vj = &data.ntm.vehicle_journeys[connection.dated_vj.vj_idx];
+    let route = &data.ntm.routes.get(&vj.route_id);
+    let line_ref = route
+        .map(|r| r.line_id.clone())
+        .unwrap_or_else(|| "".to_owned());
```

Comment:
With get_corresponding, you can have the line(s) from the vjs directly (as a ptref request)

Reply:
oh great, I forgot this!

thanks I'll use it tomorrow